### PR TITLE
Automatically adjust scoreGameWeek based on events

### LINKED ISSUE #95: Make event points increment scoreGameWeek and score for contestants.

### PR DESCRIPTION
**Pull request by tormiss**
Backend for automatically adjusting scoreGameWeek when events are committed and reverted.

Added a script that can be run along with the transfer script to set scoreGameWeek back to 0 every new game-week.

This fixes #95

### CODE REVIEW

File: admin/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+RUN npm install
+
+# Get source files
+ADD . /code/admin/
+
+# Expose port
+EXPOSE 4000
+
+# Start
+CMD ["npm", "start"]
```
Comment (Khrall) [inline admin/Dockerfile]: Tror ikke vi trenger denne filen :)

File: admin/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+version: '2'
+services:
+   web:
+     build: .
+     volumes:
+        - ".:/code/admin"
+        - "/code/admin/node_modules"
+     ports:
+        - "4000:4000"
```
Comment (Khrall) [inline admin/docker-compose.yml]: ..eller denne 😛

File: api/package.json [hunk 3]
```diff
@@ -10,7 +10,8 @@
     "load-db": "babel-node scripts/load-db.js",
     "build": "babel src --out-dir build",
     "transfers": "babel-node scripts/run-transfers.js",
-    "award-points": "babel-node scripts/run-award-points.js"
+    "award-points": "babel-node scripts/run-award-points.js",
+    "reset-scoreGameWeek": "babel-node scripts/reset-scoreGameWeek.js"
```
Comment (Khrall) [inline api/package.json]: kunne vi kanskje kalt den reset-gameweek-score? Helst ikke camelcase in any case 🐫❌

Comment (Khrall) [inline api/package.json]: Kan beholde scoreGameWeek i modellen

File: api/scripts/reset-scoreGameWeek.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  .then(results => {
+    console.log(results);
+    console.log('completed.');
+    return process.exit(0);
+  })
+  .catch(err => {
+    console.error(err);
+    process.exit(1);
+  });
+
```
Comment (Khrall) [inline api/scripts/reset-scoreGameWeek.js]: Awesome 😏

File: api/src/routes/admin/events.js [hunk 5]
```diff
@@ -114,13 +116,15 @@ module.revert = (req, res) => {
...
       })
     );
 
     const loserTasks = pointLosers.map(loser =>
       Contestant.findById(loser).then(contestant => {
         contestant.score -= loserPoints;           // eslint-disable-line
+        contestant.scoreGameWeek -= loserPoints;   // eslint-disable-line
         return contestant.save();
       })
     );
```
Comment (Khrall) [inline api/src/routes/admin/events.js]: Looks good 🤡

**Comment by tormiss** [comment]
@Khrall Can you take a quick new check on this one? Removed the unnecessary files and corrected camel-case issue.

**Pull request merged**